guides/common/modules/proc_using-ansible-provider-for-package-and-errata-actions.adoc

```diff
@@ -0,0 +1,17 @@
+[id="Using_Ansible_Provider_for_Package_and_Errata_Actions_{context}"]
+= Using Ansible provider for package and errata actions
+
+By default, {Project} is configured to use the Script provider templates for remote execution jobs.
+If you prefer using Ansible job templates for your remote jobs, you can configure {Project} to use them by default for remote execution features associated with them.
+
+.Prerequisites
+* Use remote execution in `ssh` mode.
```

Comment:
```suggestion
By default, {Project} is configured to use the Script provider templates for remote execution jobs.
If you prefer using Ansible job templates for your remote jobs, you can configure {Project} to use them by default for remote execution features associated with them.
This change can only apply to remote execution in `ssh` mode.
```
I don't think this is an actual prerequisite. But I'm not sure about the wording here.

Reply:
How is prerequisite defined? If you haven't configured your smart proxy and hosts for `ssh` mode, that means your hosts will not have the SSH keys installed and will never receive the REX job.

Reply:
Well, sure, but you don't need it in order to perform the following procedure, do you?

Reply:
I suppose you could still set the default templates for those features, yes. But then things would stop working.

Reply:
How about just a Note, then?

```
Remember that Ansible job templates only work when remote execution is configured for `ssh` mode.`
```

Reply:
That was the idea.